Web/Scotty/Trans.hs

```diff
@@ -125,5 +125,5 @@ defaultHandler f = ScottyT $ modify $ addHandler $ Just (\e -> status status500
 -- | Use given middleware. Middleware is nested such that the first declared
 -- is the outermost middleware (it has first dibs on the request and last action
 -- on the response). Every middleware is run on each request.
-middleware :: Middleware -> ScottyT e m ()
+middleware :: Monad m => Middleware -> ScottyT e m ()
```

Comment:
Again, this type is now different. I would imagine that most existing `scotty` code would continue to work after this change, since `m` is almost always a `Monad` in practice, but better to be on the safe side and document this in the changelog.